terms.md

```diff
@@ -30,7 +30,7 @@ Skip to:
 
 ## <a id="permission">Your Permission to Use the Website</a>
 
-Subject to these terms, the company gives you permission to use the website.  That permission isn't exclusive to you, and you can't transfer it to anyone else.  Others need to agree to these terms for themselves to use the website.
+Subject to these terms, the company gives you permission to use the website.  That permission is exclusive to you, and you can't transfer it to anyone else.  Others need to agree to these terms for themselves to use the website.
```

Comment:
I'll send a PR further clarifying these sentences.

"Exclusive" has more meanings in plain language. Here, I meant _nonexclusive_, as in _nonexclusive license_: not exclusive to the person receiving the permission. But it could also be read to make the permission given under the terms _exclusively_ to the user agreeing, not assignees.

Reply:
I read as the latter, but the PR cleaned up the intent; bonus -- it also has fewer words! :)